### ISSUE TITLE
Could not locate compile-time value

### ISSUE THREAD

**Issue by ultra-gitsome** [niner/Inline-Perl5#99]
The following code:
use Inline::Perl5;
use POE:from&lt<Perl5&gt>;
use lib 'C:\temp_perl6\src_working\site';

Gives this error:
===SORRY!===
Could not locate compile-time value for symbol Path

This code gives no error:
use Inline::Perl5;
use lib 'C:\temp_perl6\src_working\site';
use POE:from&lt<Perl5&gt>;

**Comment by niner** [comment]
The reason for this is, that POE exports a lot of packages including one called "IO" which then shadows the Perl 6 IO package which contains IO::Path in your current scope. Of course you have already a good workaround for that, but in other cases something like this may be useful:

```
use Inline::Perl5;
my \POE-Kernel := do {
    use POE:from<Perl5>;
    use POE::Kernel:from<Perl5>;
    POE::Kernel
}
use lib '/tmp';

POE-Kernel.run;
```

I guess in the long term I'm gonna need to add a way for the user to specify which other modules should be imported.

**Comment by ultra-gitsome** [comment]
Thanks, niner! I'm not sure if using POE under Perl 6 is the best approach for me. Like you noted, it has a lot of packages so it is relatively heavy. I have been drafting a "tiny" verision (POE6::Tiny). Using the P6 channel and react methods seems to reduce many of the bookkeeping requirements. Converting IO methods from Perl 5 to Perl 6 may simplify the code, too.

Ok, that was a bit of a digression, but I'm finding that I need to do quite a lot of process refactoring to move from P5 to P6. That refactoring seems to be valuable, but is a big effort. So obviously if we can reuse P5 code without a big performance hit, then that is a good thing.

**Comment by niner** [comment]
This was fixed by commit 00ee3d965fde2975ef758125e9d1ff6803d77517 "BREAKING: only wrap packages that are in the loaded module's namespace"
